redirects.yaml

```diff
@@ -123,11 +123,22 @@ category/product/landscape/?: https://landscape.canonical.com/
 category/product/(server|ubuntu-server-edition)/?: /server
 category/product/ubuntu-advantage/?: /support
 category/product/ubuntu-light/?: http://lubuntu.net/
+# old certification.ubuntu.com redirects
 certified/desktops/?: "/certified?category=Desktop"
 certified/laptops/?: "/certified?category=Laptop"
 certified/servers/?: "/certified?category=Server"
 certified/devices/?: "/certified?category=Device"
 certified/socs/?: "/certified?category=SoC"
+certified/desktop/?: "/certified?category=Desktop"
+certified/server/?: "/certified?category=Server"
+certified/iot/?: "/certified?category=Device"
+certified/soc/?: "/certified?category=SoC"
+certified/hardware/(?P<canonical_id>.*): "/certified/{canonical_id}"
+certified/components/(?P<component_id>.*): "/certified/component/{component_id}"
+certified/catalog/search/?: "/certified"
+certified/desktop/models/?: "/certified"
```

Comment:
Shouldn't this go to `"/certified?category=Desktop"`?

Reply:
Ok, I think the reason why I did this is because on certificaiton.ubuntu.com/desktop/models all links have `?category=Desktop`